<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="settings_btn">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>530</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: linear-gradient(130deg, rgba(59,58,72,0.6646557548800771) 0%, rgba(67,61,57,0.9279610770089286) 42%, rgba(141,129,113,1) 71%);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="source.qrc">
      <normaloff>:/images/images/settings.png</normaloff>:/images/images/settings.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 30px;
font-weight: 600;</string>
    </property>
    <property name="text">
     <string>Прогнозирование</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>571</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>35</number>
     </property>
     <item>
      <widget class="QPushButton" name="add_data_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#add_data_btn{
	font-size: 18px;
	color: rgb(0, 204, 255);	
	border-radius: 12px;
	background-color: white;
}

QPushButton:hover#add_data_btn{
	background-color: rgb(230, 250, 255);
}</string>
       </property>
       <property name="text">
        <string>Ввести данные</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="predict_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#predict_btn{
	font-size: 18px;
	color: rgb(0, 204, 255);	
	border-radius: 12px;
	background-color: white;
}

QPushButton:hover#predict_btn{
	background-color: rgb(230, 250, 255);
}</string>
       </property>
       <property name="text">
        <string>Сделать прогноз</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="features_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#features_btn{
	font-size: 18px;
	color: rgb(0, 204, 255);	
	border-radius: 12px;
	background-color: white;
}

QPushButton:hover#features_btn{
	background-color: rgb(230, 250, 255);
}</string>
       </property>
       <property name="text">
        <string>Характеристики</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>20</y>
      <width>131</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font-size: 25px;
font-weight: 550;</string>
       </property>
       <property name="text">
        <string>Результат:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="result_tbx">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-weight: 400;
font-size: 20px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="plot_lbl">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>641</width>
      <height>481</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
